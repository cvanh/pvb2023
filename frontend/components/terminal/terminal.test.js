import renderer from "react-test-renderer";
import Terminal from "./terminal";

describe("terminal should work", () => {
  it("should match the snapshot", () => {
    const component = renderer.create(
      <Terminal sdout={["$ echo kaas", "blah blah blah"]} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
