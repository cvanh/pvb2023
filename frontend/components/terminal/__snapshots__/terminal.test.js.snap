// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`terminal should work should match the snapshot 1`] = `
[
  <div>
    <p>
      $ echo kaas
    </p>
    <p>
      blah blah blah
    </p>
  </div>,
  <p>
    <input
      type="text"
    />
  </p>,
]
`;
